app/src/main/res/values/colors.xml

```diff
@@ -32,4 +32,6 @@
     <color name="color_blue_light">#B2C9D0</color>
     <color name="color_white_light">#ECEDE8</color>
     <color name="color_red_light">#E77E7A</color>
+    <color name="color_blue_mos">#01579B</color>
+    <color name="color_blue_light_transparent">#E1F5FE</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_blue_light_transparent` appears to be unused
 Unused resources make applications larger and slow down builds.